statistics/table.go

```diff
@@ -68,6 +70,113 @@ type Table struct {
 	// and the schema of the table does not change, we don't need to load the stats for this
 	// table again.
 	TblInfoUpdateTS uint64
+
+	IsPkIsHandle bool
+}
+
+// ColAndIndexExistenceMap is the meta map for statistics.Table.
+// It can tell whether a column/index really has its statistics. So we won't send useless kv request when we do online stats loading.
+type ColAndIndexExistenceMap struct {
```

Comment:
btw, shouldn't it be named `ColAndIdxExistenceMap` or `ColumnAndIndexExistenceMap`?